### ISSUE TITLE
CYPRESS@9.3.1 - Does not load UI

### ISSUE THREAD

**Issue by mmonteiroc** [cypress-io/cypress#19782]
### Current behavior

After installing cypress@9.3.1 in our project, the cypress UI does not work anymore, gets in a infinite loading (been waiting more than 10 minutes too open)

We were in the version 9.0.0 before. I tried to delete node_modules and install again, but problem persists.
![image](https://user-images.githubusercontent.com/28816737/150297945-58e05558-2925-4a94-a790-053089c1786b.png)
![image](https://user-images.githubusercontent.com/28816737/150297966-53f462ef-9f3f-47ec-b7c6-4bcc8b5b0232.png)

### Desired behavior

Cypress opens properly

### Test code to reproduce

There is no tests, as the app directly does not open at all

### Cypress Version

9.3.1

**Comment by aryzing** [comment]
Not sure if related, but just created a blank project, clean install, and it does start. 

However, it's failing to realize that it's the first time it's being started and does not scaffold all the folders + example files. Instead, just getting an error about missing plugins file.

![image](https://user-images.githubusercontent.com/4924405/150325500-7b2c8d52-a198-4471-8404-0dfa90942b10.png)

**Comment by tbiethman** [comment]
@mmonteiroc Can you give me some details about your environment?

* Windows Version
* Node version

Does `cypress run` execute your tests successfully?

**Comment by mmonteiroc** [comment]
Sorry for not having answered before:
After deleting node_modules, worked fine.

In case you want to investigate further.

My versions:
Node
![image](https://user-images.githubusercontent.com/28816737/151004128-717246c8-d03b-4e24-96e2-8504d39e90a4.png)


Windows 10
![image](https://user-images.githubusercontent.com/28816737/151004384-9977eafe-6164-4cc5-b23e-0fe6fcdb16f9.png)

**Comment by tbiethman** [comment]
@mmonteiroc Just want to confirm, your issue was resolved after you performed a re-install?

**Comment by mmonteiroc** [comment]
Yes, confirmed. But only once doing a forced re-install (node_modules deleted)

My other colleagues had the same issue when passing from old version to newer. Got fixed when re-installing :)

**Comment by fusionshoes** [comment]
Related: I'm getting failures for tests on versions beyond 9.1.1. I've tried deleting `/node_modules` and running `npm install` for every version since (or installing via `npm install --save-dev cypress@v#`). Cypress UI does open, but all tests fail. 🤷‍♂️ 
I'm on node v14.18.0.

**Comment by tbiethman** [comment]
@fusionshoes I'm going to close this issue due to the original post being resolved. Please log a separate issue following the provided template for more investigation.
